### ISSUE TITLE
Failed tests (12208): TestAsyncSchemaChanger 

### ISSUE THREAD

**Issue by tschottdorf** [cockroachdb/cockroach#4107]
The following test appears to have failed:

[#12208](https://circleci.com/gh/cockroachdb/cockroach/12208):

```
127.0.0.1:0  16:34:12.100900  21.464µs    ·meta descriptor lookup       kv/dist_sender.go:565
127.0.0.1:0  16:34:12.100951  1.837287ms  ·sending RPC                  kv/dist_sender.go:422
127.0.0.1:0  16:34:12.101052  0           ··sending to 127.0.0.1:33055  kv/send.go:236

I0203 16:34:12.105416 537 storage/engine/rocksdb.go:137  closing in-memory rocksdb instance
--- FAIL: TestAsyncSchemaChanger (3.35s)
	<autogenerated>:30: remove /tmp/TestLogic_ca: no such file or directory
	<autogenerated>:30: remove /tmp/TestLogic_cert: no such file or directory
	<autogenerated>:30: remove /tmp/TestLogic_key: no such file or directory
=== RUN   TestSplitOnTableBoundaries
I0203 16:34:12.119196 537 rpc/clock_offset.go:153  monitoring cluster offset
I0203 16:34:12.126557 537 server/node.go:318  store store=0:0 ([]=) not bootstrapped
I0203 16:34:12.129822 537 raft/raft.go:446  [group 1] 1 became follower at term 5
I0203 16:34:12.129973 537 raft/raft.go:234  [group 1] newRaft 1 [peers: [1], term: 5, commit: 10, applied: 10, lastindex: 10, lastterm: 5]
I0203 16:34:12.130055 537 raft/raft.go:526  [group 1] 1 is starting a new election at term 5
I0203 16:34:12.130132 537 raft/raft.go:459  [group 1] 1 became candidate at term 6
--
--- PASS: TestColumnTypeSQLString (0.00s)
	<autogenerated>:28: prior leak detected, leaktest disabled
=== RUN   TestInitialKeys
--- PASS: TestInitialKeys (0.01s)
	<autogenerated>:28: prior leak detected, leaktest disabled
FAIL
FAIL	github.com/cockroachdb/cockroach/sql	93.396s
=== RUN   TestNoLinkForbidden
--- PASS: TestNoLinkForbidden (1.81s)
=== RUN   TestDates
I0203 16:32:32.334995 525 rpc/clock_offset.go:153  monitoring cluster offset
I0203 16:32:32.353332 525 server/node.go:318  store store=0:0 ([]=) not bootstrapped
I0203 16:32:32.360742 525 raft/raft.go:446  [group 1] 1 became follower at term 5
I0203 16:32:32.361180 525 raft/raft.go:234  [group 1] newRaft 1 [peers: [1], term: 5, commit: 10, applied: 10, lastindex: 10, lastterm: 5]
I0203 16:32:32.361422 525 raft/raft.go:526  [group 1] 1 is starting a new election at term 5
I0203 16:32:32.361672 525 raft/raft.go:459  [group 1] 1 became candidate at term 6
I0203 16:32:32.361902 525 raft/raft.go:508  [group 1] 1 received vote from 1 at term 6

```
Please assign, take a look and update the issue accordingly.

**Comment by vivekmenezes** [comment]
I have no clue as to how to debug this with no logs. I don't even know where it failed! Is there a way to get the test logs from CI?

**Comment by tamird** [comment]
https://circle-artifacts.com/gh/cockroachdb/cockroach/12208/artifacts/1/tmp/circle-artifacts.RlofbuQ/testrace.log

**Comment by tamird** [comment]
Fixed in #4166

**Comment by vivekmenezes** [comment]
ah ha good to know. thanks
